internal/quickstart/toggle_flag.go

```diff
@@ -94,6 +97,9 @@ func (m toggleFlagModel) View() string {
 
 	if m.flagWasEnabled {
 		furtherInstructions = fmt.Sprintf("\n\nCheck your %s to see the change!", logTypeMap[m.sdkKind])
+		if m.alreadyEnabled {
+			furtherInstructions = "\n\nFlag was toggled too quickly."
```

Comment:
Removed toggle state. Also followed the example from line 70 `Error toggling flag: you have toggled too quickly.`